dragonfly/status_log_handlers/amqp_handler.py

```diff
@@ -42,16 +42,18 @@ def update_parser(self, parser):
 
     def setLevel(self, level):
         '''
-        for now, force the to critical...
+        for now, force the to warning...
         '''
-        if level != logging.CRITICAL:
-            print('warning: slack is only ever critical, setting that')
+        if level < logging.WARNING:
+            print('warning: slack is only ever warning, setting that')
```

Comment:
did you mean to also call `super(AMQPHandler, self).seLevel(logging.WARNING)` after this print?